thoth/user_api/api_v1.py

```diff
@@ -399,6 +401,129 @@ def list_python_package_indexes():
     return graph.python_package_index_listing()
 
 
+def post_build(
+    log_info: dict,
+    base_image: str = None,
+    image: str = None,
+    debug: bool = False,
+    registry_user: str = None,
+    registry_password: str = None,
+    environment_type: str = None,
+    origin: str = None,
+    verify_tls: bool = True,
+    force: bool = False,
+    output_format: str = "json",
+    handler: str = None,
+    limit: int = None,
+):
+    """Run analysis on a build."""
+    response = {"base_image_analysis": {}, "output_image_analysis": {}, "buildlog_analysis": {}}
+    status = 202
+    if image:
+        # Run image analysis
+        image_analyze_response, image_analyze_status = post_analyze(
+            image=image,
+            debug=debug,
+            registry_user=registry_user,
+            registry_password=registry_password,
+            environment_type=environment_type,
+            origin=origin,
+            verify_tls=verify_tls,
+            force=force,
+        )
+        if image_analyze_status == 202:
+            response["output_image_analysis"] = image_analyze_response
+        else:
+            status = image_analyze_status
```

Comment:
Handling it like this can be dangerous - it will also introduce troubles on schema and parsing response in automatically generated swagger client. As results are cached, you can try to schedule both image analyses and return error code if any of them fails. Otherwise, create separate records for both of them in the resulting dict as you did.